connector_easypost_tracker/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================
+EasyPost Connector Tracker
+==========================
+
+This module provides EasyPost Tracking functionality.
```

Comment:
This module provides web hooks to allow EasyPost to update package tracking status in Odoo.